docker-compose/_posts/2017-07-05-building-a-compose-file-for-runnable.md

```diff
@@ -0,0 +1,53 @@
+---
+title: Building a Compose File for Runnable
```

Comment:
Should just be "Building a Compose File", I think. I think you're trying to say the Compose file rules are specific to Runnable, but for users who aren't already aware, it just seems redundant to say "…for Runnable"